tutorials/dataloader/dataloader-image-data.jl

```diff
@@ -1,48 +1,80 @@
+# # An example of DataLoader using image data
+
+
+# In this example, we show how to load image data in Flux DataLoader and process it in mini-batches. 
+# We use the [DataLoader](https://fluxml.ai/Flux.jl/stable/data/dataloader/#Flux.Data.DataLoader) type 
+# to handle iteration over mini-batches of data. 
+# Moreover, we load the [MNIST dataset](https://juliaml.github.io/MLDatasets.jl/stable/datasets/MNIST/) 
+# using the [MLDatasets](https://juliaml.github.io/MLDatasets.jl/stable/) package.
+ 
+
+# Load the packages we need:
+
 using MLDatasets: MNIST
 using Flux.Data: DataLoader
 using Flux: onehotbatch
 using Parameters: @with_kw
 
+# We set a default value for the size of the mini-batches:
+
 @with_kw mutable struct Args
-    minibath_size::Int = 128  # Size of mini-batch
+    minibath_size::Int = 128  ## Size of mini-batch
 end
 
+# ## Get the data
+ 
+# We create the function `get_data` to get, preprare and load the data onto a DataLoader object.
+
 function get_data(args)
 
-    # Load the MNIST train and test data from MLDatasets
+    ## Load the MNIST train and test data from MLDatasets
     train_x, train_y = MNIST.traindata(Float32)
     test_x, test_y = MNIST.testdata(Float32)
 
-    # Reshape data to 28x28x1 multi-dimensional array
+    ## Reshape data to 28x28x1 multi-dimensional array
```

Comment:
Question about the double `##`: are these going to be used for some kind of literate rendered output later?

Reply:
@ToucheSir that's right, we are adding these so we can use Literate to convert the file to markdown and notebooks